workspace/__lib__/xod/bits/i32-to-number/patch.xodp

```diff
@@ -0,0 +1,59 @@
+{
+  "description": "Converts four bytes into integer-valued number. Result of the conversion will be between -2147483647.00 and 2147483647.00.",
```

Comment:
An important point about the sign is a bit implicit. Also, are you sure that the lower bound is 214748364**7** and not 214748364**8**? Also, one article is missing.

> Converts 32-bit signed integer presented as four bytes to a number. A result of the conversion will be between -2147483648.00 and 2147483647.00.